java-diff-utils/src/main/java/com/github/difflib/patch/AbstractDelta.java

```diff
@@ -74,7 +74,10 @@ protected VerifyChunk verifyAntApplyTo(List<T> target) throws PatchFailedExcepti
      * Apply patch fuzzy.
      *
      * @param target the list this patch will be applied to
-     * @param fuzz the count of elements which ignores prefix and suffix
+     * @param fuzz the count of first/end elements which is not matched.
```

Comment:
I didn't understand the concept of fuzzy patch before reading this Javadoc and I was unable to understand it. I don't think this is your fault. I don't think we'll be able to explain the parameters unless the user understands the purpose/meaning of `fuzzy patch` to begin with.

I suggest replacing the parameter Javadoc with something along the lines of: `@param fuzz the number of elements to ignore before/after the patched elements"` and adding `@see <a href="https://www.gnu.org/software/diffutils/manual/html_node/Inexact.html">Description of Fuzzy Patch</a> for more information`.